cypress/integration/27-expenses.create.test.js

```diff
@@ -21,7 +21,7 @@ const uploadReceipt = (dropzoneElement = '.InputTypeDropzone') => {
 describe('new expense when logged out', () => {
   it('requires to login to submit an expense', () => {
     cy.visit('/testcollective/expenses');
-    cy.get('.Button.submitExpense.darkBackground')
+    cy.get('.a.submitExpense.darkBackground')
```

Comment:
Here, you don't want a "class selector" `.a` but an "element selector" `a`.

Reply:
Thanks a lot, @znarf. Updating corrections

Reply:
Please, I'm getting a new error.

Reply:
I'm restarting the job, maybe a random error, doesn't seems related with the changes.